### ISSUE TITLE
Chinese character does not show up

### ISSUE THREAD

**Issue by johntheocs** [Inlyne-Project/inlyne#149]
I am wondering if this issue is related to font setting or rendering. Thanks for this amazing work.

<img width="130" alt="Screenshot 2023-06-27 at 7 11 29 PM" src="https://github.com/trimental/inlyne/assets/115430128/2d74cac2-8b59-43da-bf84-7a3ddcf906c1">

**Comment by CosmicHorrorDev** [comment]
Have you tried using the latest `main`? There are unreleased changes that add in font fallback support

**Comment by johntheocs** [comment]
Yes, it works with the latest `main`. But the font is a little strange, different from other applications.

**Comment by CosmicHorrorDev** [comment]
That could be up to whatever font ends up getting selected. You can set your default regular font in a config file that's described a bit here

https://github.com/trimental/inlyne#configuration

**Comment by johntheocs** [comment]
<img width="202" alt="Screenshot 2023-06-27 at 7 28 05 PM" src="https://github.com/trimental/inlyne/assets/115430128/9a1c9c96-1098-4eaf-b69c-2208a1680f41">

Weird. Problems with the bold text, but the normal text is fine.

**Comment by CosmicHorrorDev** [comment]
Hmmm, that looks like it may be selecting different fonts for different characters? That does seem really odd

We're currently using a fork of `cosmic-text` to handle font stuff, so it's possible that the issue will get resolved as `cosmic-text` matures

**Comment by johntheocs** [comment]
Yes, there is an open issue there: https://github.com/pop-os/cosmic-text/issues/114

I am using Zed editor, which is GPU powered too, but does not have this type of issue.

**Comment by johntheocs** [comment]
Lapce editor is using a forked version of cosmic-text: https://github.com/lapce/cosmic-text, which works well.

**Comment by CosmicHorrorDev** [comment]
You can try adding that repo as a patch to the `Cargo.toml` file and see if it resolved the issue. We'll need an actual published version next time we go to cut a release though
